app/components/StatsSection/StatsSection.tsx

```diff
@@ -44,19 +43,18 @@ const StatsSection = ({
 
         {/** Events Graph and Most Active area */}
         <div className="flex flex-col gap-4">
-          <div className="flex flex-col items-center p-4 bg-gray-100 rounded-lg shadow-inner gap-4">
+          <div className="flex flex-col items-center p-4 bg-gray-100 rounded-lg shadow-inner sm:w-[100%] md:w-[325px] lg:w-[400px] xl:w-[100%]">
             <p className="text-md font-medium text-gray-500">
               Events Throughout {isWeekly ? 'Week' : 'Day'}
             </p>
-            <EarthquakeChart />
-            {/* <div className="bg-blue-500 w-[100%] h-[300px]" /> */}
+            <div style={{ width: '100%', height: '325px' }}>
```

Comment:
I suspect it's easier than this to make this graph responsive - the docs are a little confusing for nivo, but I'll try and fix this in the future. But a TODO should be added in the meantime.

Reply:
Also remove that stand in blue box